package.json

```diff
@@ -10,6 +10,8 @@
     "prettier": "prettier '**/*.{js?(on),ts?(x),md}' --ignore-path .gitignore --write --list-different",
     "eslint": "eslint '**/*.{ts,tsx}' --ignore-path .gitignore",
     "eslint-fix": "npm run eslint -- --fix",
+    "check-types": "tsc --noemit",
```

Comment:
Sneaking this in here mainly to help myself, but I can move to a separate PR if folks feel strongly about the scope creep.

Reply:
:+1: - this is a great idea.